packages/discovery/src/discovery/colorize/colorize.ts

```diff
@@ -25,7 +26,7 @@ export function colorize(
   for (const e of structure.entries) {
     const entryConfig = makeEntryColorConfig(
```

Comment:
I'm not sure why makeEntryColorConfig should receive on `EthereumAddress`. I'ts not used outside discovery. It should accept ChainSpecificAddress type.